test/transcoding/FPGAInfiniteLoop.ll

```diff
@@ -0,0 +1,178 @@
+; RUN: llvm-as %s -o %t.bc
```

Comment:
I'd suggest commenting out the source code and providing the link to the intel/llvm project, which apparently was used to produce the LLVM IR (the same way it was done for test/IntelFPGAReg.ll)